Frameworks/Social.Xaml/ShareSheet.xaml

```diff
@@ -0,0 +1,20 @@
+<ContentDialog
+    x:Class="Social.Xaml.ShareSheet"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Social.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Title="Facebook"
+    PrimaryButtonText="Post"
+    SecondaryButtonText="Cancel"
```

Comment:
Hang on, let me search the web for "XAML property binding"...

OK, I see what you mean. Fair point on the TextBlocks and such.

I'm not sure I see the harm in hard-coding the titles of the Post and Cancel buttons, though. Those won't ever change, so I don't see a need to complicate things.

---
In reply to: [107995907](https://github.com/Microsoft/WinObjC/pull/2314#discussion_r107995907) [](ancestors = 107995907,107995837)